pkg/crc/oc/oc.go

```diff
@@ -0,0 +1,30 @@
+package oc
+
+import (
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+type OC struct {
```

Comment:
Not familiar with go conventions, should be this `type OC` or `type Oc`?

Reply:
@cfergeau we can have it `Oc`, shouldn't be problem, I put it both capital since both are initial and doesn't look good if one is small and one is capital one :)